frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/propertiesList.component.tsx

```diff
@@ -24,26 +24,33 @@ import { useFormContext } from 'react-hook-form';
 import { TicketProperty } from './properties/properties.helper';
 import { UnsupportedProperty } from './properties/unsupportedProperty.component';
 import { ErrorTextGap, PropertiesListContainer } from './ticketsForm.styles';
+import { SequencingProperties } from '../tickets.constants';
 
 interface PropertiesListProps {
 	properties: PropertyDefinition[];
 	module: string;
 	onPropertyBlur?: (...args) => void;
 }
 
+const isSequencingProperty = (inputName: string) => [
+	`modules.sequencing.${SequencingProperties.START_TIME}`,
+	`modules.sequencing.${SequencingProperties.END_TIME}`,
```

Comment:
maybe we can move SEQUENCING_START_TIME and SEQUENCING_END_TIME from sequencing property and use that here as well as in sequencing property